src/ts/component/popup/page/membership/paid.tsx

```diff
@@ -49,6 +50,11 @@ const PopupMembershipPagePaid = observer(class PopupMembershipPagePaid extends R
 
 		let periodText = '';
 		let labelText = '';
+		let paidOnOtherPlatform = false;
+
+		if (membership.tier == I.TierType.Builder && membership.paymentMethod != I.PaymentMethod.Stripe) {
```

Comment:
Deconstruct membership properties
`const { tier, paymentMethod } = membership`
Add parenthesis for clause parts `((tier == I.TierType.Builder) && (paymentMethod != I.PaymentMethod.Stripe))`